pyemma/coordinates/io/feature_reader.py

```diff
@@ -157,7 +157,13 @@ def dimension(self):
 
         :return:
         """
-        return self.featurizer.dimension()
+        if len(self.featurizer.active_features)==0:
+            # special case: cartesion coordinates
+            top = mdtraj.load(self.topfile)
```

Comment:
@fabian-paul you can reuse the topology which have already been loaded in the featurizer. After this I'm happy to merge your great work!